Cargo.toml

```diff
@@ -8,3 +10,14 @@ members = [
     "state-indexer",
     "tx-indexer",
 ]
+
+[patch.crates-io]
+near-chain-configs = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+near-crypto = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+near-jsonrpc-primitives = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+near-indexer-primitives = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+near-primitives = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+near-vm-runner = { git = 'https://github.com/near/nearcore.git', branch = '1.36.0' }
+
+near-jsonrpc-client = { git = 'https://github.com/kobayurii/near-jsonrpc-client-rs.git', branch = '0.7.0' }
```

Comment:
Add comments around these two dependencies explaining why we use your forks instead of going through the PRs process in those projects, please.